test/CppTests/debuggees/natvis/src/visualizer_files/Simple.natvis

```diff
@@ -31,6 +31,16 @@
   </Expand>
   </Type>
 
+  <Type Name="SimpleArrayPointer">
```

Comment:
There should be no need for a natvis type for testing the `<variableName>,<size>` format. 

I would recommend testing on a regular int pointer pointing to a block of memory then evaluating the format specifier on it.